common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -319,6 +319,32 @@ class DB(contextWrapper: ContextWrapper) {
     )
   }
 
+  def userEventsDates(): List[String] = {
+    contextWrapper.dbContext.run(
+      quote(
+        query[UserEventDB].schema(_.entity("user_events")).map(event => event.event_datetime)
+      )
+    ).map(datetime => {
```

Comment:
Here `.map(datetime => {` the bracket is not necessary and can be written as `.map { datetime =>`